packages/core-data/src/hooks/test/use-resource-permissions.js

```diff
@@ -24,11 +24,9 @@ describe( 'useResourcePermissions', () => {
 		registry.register( coreDataStore );
 
 		triggerFetch.mockImplementation( () => ( {
-			headers: {
-				get: () => ( {
-					allow: 'POST',
-				} ),
-			},
+			headers: new Headers( {
+				allow: 'POST',
+			} ),
```

Comment:
[`Headers`](https://developer.mozilla.org/en-US/docs/Web/API/Headers/Headers) is the right interface for dealing with response headers. Just using the constructor adds the necessary functionality, including proper support for `.get()`.